android-wrapper/.gitignore

```diff
@@ -0,0 +1,15 @@
+*.iml
```

Comment:
This Android wrapper is entirely generated code, right? I usually don't check in generated code.